### PR TITLE
Add new "Games!" (legacy) discovery module

### PR DESCRIPTION
**Pull request by diox**
Also introduce a new bg that is not used yet and optimize disco backgrounds while we're at it.

### CODE REVIEW

File: src/olympia/legacy_discovery/models.py [hunk 1]
```diff
@@ -20,3 +20,6 @@ class DiscoveryModule(ModelBase):
     class Meta:
         db_table = 'discovery_modules'
         unique_together = ('app', 'module')
+
+    def __unicode__(self):
```
Comment (muffinresearch) [inline src/olympia/legacy_discovery/models.py]: Does this need some test coverage or will it be covered implcitly?

**Comment by muffinresearch** [comment]
lgtm r+ if green

**Comment by muffinresearch** [comment]
I'll wait for this before tagging.

File: src/olympia/legacy_discovery/models.py (hunk 1 above)
Comment (diox) [inline src/olympia/legacy_discovery/models.py]: It's really just for convenience when debugging, not used anywhere.

**Pull request merged**